my-app/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import logo from './logo.svg';
+import Board from './Board';
+import './App.css';
+
+export const BoardSquareContext = React.createContext();
```

Comment:
Nice - I like the choice to use a context here. Since these both have to do with the board, I'd probably make one `BoardContext` that would then have multiple properties on it - one for the board array and one for the `setBoard` function. 

```jsx
<BoardContext.Provider value={{board: board, setBoard: setBoard }}
```